Mathlib/Topology/UniformSpace/Matrix.lean

```diff
@@ -27,7 +27,8 @@ instance instUniformAddGroup [AddGroup 𝕜] [UniformAddGroup 𝕜] :
 
 theorem uniformity :
     𝓤 (Matrix m n 𝕜) = ⨅ (i : m) (j : n), (𝓤 𝕜).comap fun a => (a.1 i j, a.2 i j) := by
-  erw [Pi.uniformity]
+  unfold instUniformSpace Matrix
```

Comment:
`unfold Matrix` seems pretty criminal; I think the right fix here is to induce the uniform space through `Matrix.of` on line 22?